analyzer/mfcckeeper.cc

```diff
@@ -25,6 +25,9 @@
 #include <torch/MemoryDataSet.h>
 #include <torch/DiagonalGMM.h>
 
+#undef max
```

Comment:
:) Given that this is C++ code and we have std::max/std::min, I think undef'ing is a good idea. However, could you add a comment indicating that? Something like:

/* Undef to prevent collisions with std versions of the same. */